2022/02/24 09:49:22 [crit] 27121#27121: *1 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 24.48.113.24, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 09:49:22 [error] 27121#27121: *1 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 24.48.113.24, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 09:49:22 [crit] 27121#27121: *1 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 24.48.113.24, server: 3.14.141.61, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/favicon.ico", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 09:49:22 [error] 27121#27121: *1 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 24.48.113.24, server: 3.14.141.61, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/favicon.ico", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 09:58:05 [error] 27121#27121: *5 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 24.48.113.24, server: 3.14.141.61, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 10:02:18 [error] 27121#27121: *8 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 24.48.113.24, server: 3.14.141.61, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com", referrer: "http://ec2-3-14-141-61.us-east-2.compute.amazonaws.com/"
2022/02/24 10:24:16 [error] 27121#27121: *12 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 24.48.113.24, server: 3.14.141.61, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com", referrer: "http://ec2-3-14-141-61.us-east-2.compute.amazonaws.com/"
2022/02/24 11:10:53 [crit] 27121#27121: *17 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 23.251.102.74, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 11:10:53 [error] 27121#27121: *17 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 23.251.102.74, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 11:18:00 [crit] 27121#27121: *20 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 167.94.138.62, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61:80"
2022/02/24 11:18:00 [error] 27121#27121: *20 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 167.94.138.62, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61:80"
2022/02/24 11:18:00 [crit] 27121#27121: *22 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 167.94.138.62, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 11:18:00 [error] 27121#27121: *22 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 167.94.138.62, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 11:55:44 [crit] 27121#27121: *25 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 64.62.197.122, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 11:55:44 [error] 27121#27121: *25 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 64.62.197.122, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 12:57:40 [crit] 27121#27121: *27 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 172.96.14.34, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/24 12:57:40 [error] 27121#27121: *27 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 172.96.14.34, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/24 12:57:40 [crit] 27121#27121: *29 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 172.96.14.34, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 12:57:40 [error] 27121#27121: *29 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 172.96.14.34, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 12:59:54 [crit] 27121#27121: *31 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 93.23.14.126, server: 3.14.141.61, request: "HEAD / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 12:59:54 [error] 27121#27121: *31 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 93.23.14.126, server: 3.14.141.61, request: "HEAD / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 12:59:57 [crit] 27121#27121: *33 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 93.23.14.126, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 12:59:57 [error] 27121#27121: *33 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 93.23.14.126, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 13:11:54 [crit] 27121#27121: *35 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 205.185.125.167, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpMyAdmin/scripts/setup.php"
2022/02/24 13:11:54 [error] 27121#27121: *35 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 205.185.125.167, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpMyAdmin/scripts/setup.php"
2022/02/24 13:30:21 [crit] 27121#27121: *38 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 13:30:21 [error] 27121#27121: *38 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 13:30:21 [crit] 27121#27121: *40 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /HNAP1/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/HNAP1/", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:21 [error] 27121#27121: *40 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /HNAP1/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/HNAP1/", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:21 [crit] 27121#27121: *42 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /hudson/script HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/hudson/script", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:21 [error] 27121#27121: *42 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /hudson/script HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/hudson/script", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [crit] 27121#27121: *44 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /script HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/script", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [error] 27121#27121: *44 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /script HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/script", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [crit] 27121#27121: *46 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /sqlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/sqlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [error] 27121#27121: *46 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /sqlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/sqlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [crit] 27121#27121: *48 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /sqlitemanager/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/sqlitemanager/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:22 [error] 27121#27121: *48 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /sqlitemanager/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/sqlitemanager/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [crit] 27121#27121: *50 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLiteManager/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/SQLiteManager/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [error] 27121#27121: *50 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLiteManager/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/SQLiteManager/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [crit] 27121#27121: *52 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/SQLite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [error] 27121#27121: *52 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/SQLite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [crit] 27121#27121: *54 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/SQlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:23 [error] 27121#27121: *54 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/SQlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [crit] 27121#27121: *56 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [error] 27121#27121: *56 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [crit] 27121#27121: *58 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /test/sqlite/SQLiteManager-1.2.0/SQLiteManager-1.2.0/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/test/sqlite/SQLiteManager-1.2.0/SQLiteManager-1.2.0/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [error] 27121#27121: *58 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /test/sqlite/SQLiteManager-1.2.0/SQLiteManager-1.2.0/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/test/sqlite/SQLiteManager-1.2.0/SQLiteManager-1.2.0/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [crit] 27121#27121: *60 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLiteManager-1.2.4/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/SQLiteManager-1.2.4/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [error] 27121#27121: *60 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /SQLiteManager-1.2.4/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/SQLiteManager-1.2.4/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [crit] 27121#27121: *62 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 92.101.147.153, server: 3.14.141.61, request: "GET /agSearch/SQlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/agSearch/SQlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 13:30:24 [error] 27121#27121: *62 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 92.101.147.153, server: 3.14.141.61, request: "GET /agSearch/SQlite/main.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/agSearch/SQlite/main.php", host: "3.14.141.61", referrer: "http://3.14.141.61/"
2022/02/24 14:30:11 [crit] 27121#27121: *64 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/24 14:30:11 [error] 27121#27121: *64 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/24 14:30:12 [crit] 27121#27121: *66 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 14:30:12 [error] 27121#27121: *66 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 15:00:56 [crit] 27121#27121: *68 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 34.140.248.32, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 15:00:56 [error] 27121#27121: *68 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 34.140.248.32, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 16:29:56 [crit] 27121#27121: *71 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 209.141.52.239, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpMyAdmin/scripts/setup.php"
2022/02/24 16:29:56 [error] 27121#27121: *71 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 209.141.52.239, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpMyAdmin/scripts/setup.php"
2022/02/24 17:35:30 [crit] 27121#27121: *75 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/24 17:35:30 [error] 27121#27121: *75 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/24 17:35:31 [crit] 27121#27121: *77 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 17:35:31 [error] 27121#27121: *77 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 18:03:51 [crit] 27121#27121: *79 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 178.239.21.90, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 18:03:51 [error] 27121#27121: *79 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 178.239.21.90, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 18:30:41 [crit] 27121#27121: *85 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 45.146.164.234, server: 3.14.141.61, request: "GET / HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/"
2022/02/24 18:30:41 [error] 27121#27121: *85 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 45.146.164.234, server: 3.14.141.61, request: "GET / HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/"
2022/02/24 19:04:44 [crit] 27121#27121: *87 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 128.14.133.58, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 19:04:44 [error] 27121#27121: *87 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 128.14.133.58, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 19:20:34 [crit] 27121#27121: *89 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 209.141.61.19, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpMyAdmin/scripts/setup.php"
2022/02/24 19:20:34 [error] 27121#27121: *89 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 209.141.61.19, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpMyAdmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpMyAdmin/scripts/setup.php"
2022/02/24 20:41:39 [crit] 27121#27121: *92 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/24 20:41:39 [error] 27121#27121: *92 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/24 20:41:40 [crit] 27121#27121: *94 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 20:41:40 [error] 27121#27121: *94 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 20:51:09 [crit] 27121#27121: *96 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 72.167.47.168, server: 3.14.141.61, request: "GET /app/etc/local.xml HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/app/etc/local.xml", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com"
2022/02/24 20:51:09 [error] 27121#27121: *96 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 72.167.47.168, server: 3.14.141.61, request: "GET /app/etc/local.xml HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/app/etc/local.xml", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com"
2022/02/24 20:51:09 [crit] 27121#27121: *98 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 72.167.47.168, server: 3.14.141.61, request: "GET /magmi/conf/magmi.ini HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/magmi/conf/magmi.ini", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com"
2022/02/24 20:51:09 [error] 27121#27121: *98 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 72.167.47.168, server: 3.14.141.61, request: "GET /magmi/conf/magmi.ini HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/magmi/conf/magmi.ini", host: "ec2-3-14-141-61.us-east-2.compute.amazonaws.com"
2022/02/24 21:18:46 [crit] 27121#27121: *100 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 45.146.164.234, server: 3.14.141.61, request: "GET / HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/"
2022/02/24 21:18:46 [error] 27121#27121: *100 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 45.146.164.234, server: 3.14.141.61, request: "GET / HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/"
2022/02/24 21:45:10 [crit] 27121#27121: *102 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 209.141.35.105, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpmyadmin/scripts/setup.php"
2022/02/24 21:45:10 [error] 27121#27121: *102 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 209.141.35.105, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpmyadmin/scripts/setup.php"
2022/02/24 22:21:30 [crit] 27121#27121: *105 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 128.14.209.162, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 22:21:30 [error] 27121#27121: *105 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 128.14.209.162, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 22:39:37 [crit] 27121#27121: *107 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 128.14.209.170, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 22:39:37 [error] 27121#27121: *107 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 128.14.209.170, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/24 23:35:52 [crit] 27121#27121: *109 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/24 23:35:52 [error] 27121#27121: *109 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/24 23:35:52 [crit] 27121#27121: *111 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/24 23:35:52 [error] 27121#27121: *111 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 00:39:53 [crit] 27121#27121: *113 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 23.128.248.42, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 00:39:53 [error] 27121#27121: *113 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 23.128.248.42, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 01:20:17 [crit] 27121#27121: *115 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 64.62.197.212, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 01:20:17 [error] 27121#27121: *115 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 64.62.197.212, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 01:35:40 [crit] 27121#27121: *117 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 192.241.205.252, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 01:35:40 [error] 27121#27121: *117 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 192.241.205.252, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 01:37:08 [crit] 27121#27121: *119 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 81.38.99.77, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 01:37:08 [error] 27121#27121: *119 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 81.38.99.77, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 02:00:36 [crit] 27121#27121: *121 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 45.143.99.60, server: 3.14.141.61, request: "GET /manifest.json HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/manifest.json", host: "app.nereus.finance"
2022/02/25 02:00:36 [error] 27121#27121: *121 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 45.143.99.60, server: 3.14.141.61, request: "GET /manifest.json HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/manifest.json", host: "app.nereus.finance"
2022/02/25 02:12:41 [crit] 27121#27121: *123 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 02:12:41 [error] 27121#27121: *123 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 02:12:42 [crit] 27121#27121: *125 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 02:12:42 [error] 27121#27121: *125 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 03:25:53 [crit] 27121#27121: *127 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 175.100.20.245, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 03:25:53 [error] 27121#27121: *127 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 175.100.20.245, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 03:25:55 [crit] 27121#27121: *129 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 175.100.20.245, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 03:25:55 [error] 27121#27121: *129 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 175.100.20.245, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 04:19:53 [crit] 27121#27121: *131 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 205.185.127.43, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpmyadmin/scripts/setup.php"
2022/02/25 04:19:53 [error] 27121#27121: *131 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 205.185.127.43, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpmyadmin/scripts/setup.php"
2022/02/25 04:46:40 [crit] 27121#27121: *133 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 104.149.177.122, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 04:46:40 [error] 27121#27121: *133 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 104.149.177.122, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 04:53:37 [crit] 27121#27121: *137 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 5.188.210.227, server: 3.14.141.61, request: "GET http://5.188.210.227/echo.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/echo.php", host: "5.188.210.227", referrer: "https://www.google.com/"
2022/02/25 04:53:37 [error] 27121#27121: *137 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 5.188.210.227, server: 3.14.141.61, request: "GET http://5.188.210.227/echo.php HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/echo.php", host: "5.188.210.227", referrer: "https://www.google.com/"
2022/02/25 04:59:11 [crit] 27121#27121: *139 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 193.118.53.210, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 04:59:11 [error] 27121#27121: *139 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 193.118.53.210, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 05:21:25 [crit] 27121#27121: *141 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 05:21:25 [error] 27121#27121: *141 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 05:21:26 [crit] 27121#27121: *143 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 05:21:26 [error] 27121#27121: *143 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.217, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 06:10:43 [crit] 27121#27121: *145 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 2.56.59.96, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61:80"
2022/02/25 06:10:43 [error] 27121#27121: *145 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 2.56.59.96, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61:80"
2022/02/25 06:25:42 [crit] 27121#27121: *147 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 209.141.62.127, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/phpmyadmin/scripts/setup.php"
2022/02/25 06:25:42 [error] 27121#27121: *147 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 209.141.62.127, server: 3.14.141.61, request: "GET http://3.14.141.61:80/phpmyadmin/scripts/setup.php HTTP/1.0", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/phpmyadmin/scripts/setup.php"
2022/02/25 06:27:29 [crit] 27121#27121: *149 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 06:27:29 [error] 27121#27121: *149 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 06:27:29 [crit] 27121#27121: *151 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 06:27:29 [error] 27121#27121: *151 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 07:09:56 [crit] 27121#27121: *153 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 176.97.210.170, server: 3.14.141.61, request: "POST /boaform/admin/formLogin HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/boaform/admin/formLogin", host: "3.14.141.61:80", referrer: "http://3.14.141.61:80/admin/login.asp"
2022/02/25 07:09:56 [error] 27121#27121: *153 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 176.97.210.170, server: 3.14.141.61, request: "POST /boaform/admin/formLogin HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/boaform/admin/formLogin", host: "3.14.141.61:80", referrer: "http://3.14.141.61:80/admin/login.asp"
2022/02/25 08:00:22 [crit] 27121#27121: *156 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 08:00:22 [error] 27121#27121: *156 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 08:00:22 [crit] 27121#27121: *158 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 08:00:22 [error] 27121#27121: *158 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 08:41:04 [crit] 27121#27121: *160 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 192.241.216.169, server: 3.14.141.61, request: "GET /portal/redlion HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/portal/redlion", host: "3.14.141.61"
2022/02/25 08:41:04 [error] 27121#27121: *160 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 192.241.216.169, server: 3.14.141.61, request: "GET /portal/redlion HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/portal/redlion", host: "3.14.141.61"
2022/02/25 08:58:43 [crit] 27121#27121: *162 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 192.241.221.178, server: 3.14.141.61, request: "GET /actuator/health HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/actuator/health", host: "3.14.141.61"
2022/02/25 08:58:43 [error] 27121#27121: *162 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 192.241.221.178, server: 3.14.141.61, request: "GET /actuator/health HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/actuator/health", host: "3.14.141.61"
2022/02/25 09:09:40 [crit] 27121#27121: *164 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 09:09:40 [error] 27121#27121: *164 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
2022/02/25 09:09:42 [crit] 27121#27121: *166 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 09:09:42 [error] 27121#27121: *166 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 185.254.196.218, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 09:15:55 [crit] 27121#27121: *168 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 183.136.226.3, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61:80"
2022/02/25 09:15:55 [error] 27121#27121: *168 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 183.136.226.3, server: 3.14.141.61, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61:80"
2022/02/25 09:16:30 [crit] 27121#27121: *170 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 183.136.226.3, server: 3.14.141.61, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/favicon.ico", host: "3.14.141.61:80"
2022/02/25 09:16:30 [error] 27121#27121: *170 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 183.136.226.3, server: 3.14.141.61, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/favicon.ico", host: "3.14.141.61:80"
2022/02/25 09:24:52 [crit] 27121#27121: *172 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "3.14.141.61"
2022/02/25 09:24:52 [error] 27121#27121: *172 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "POST / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "3.14.141.61"
2022/02/25 09:24:53 [crit] 27121#27121: *174 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/.env", host: "3.14.141.61"
2022/02/25 09:24:53 [error] 27121#27121: *174 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 135.125.244.48, server: 3.14.141.61, request: "GET /.env HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/.env", host: "3.14.141.61"
